src/tasks/TaskTerminalBase.ts

```diff
@@ -14,7 +14,7 @@ export abstract class TaskTerminalBase implements vscode.Pseudoterminal {
     public onDidWrite: vscode.Event<string> = this._writeEmitter.event;
     public onDidClose?: vscode.Event<number> = this._closeEmitter.event;
 
-    public abstract async open(initialDimensions: vscode.TerminalDimensions | undefined): Promise<void>;
```

Comment:
hi thanks for this PR. are there any reason to remove this async, we are making this method synchronized ?

Reply:
Thanks for comment, ☕️🥷 basically its this: https://stackoverflow.com/questions/65355164/error-ts1243-async-modifier-cannot-be-used-with-abstract-modifier OR this -> https://github.com/microsoft/TypeScript/issues/49096 

If you see the usage of this funct you will see how it is used.